README.md

```diff
@@ -20,4 +20,28 @@
 
 <img width="200" alt="Screenshot 2023-07-10 at 7 43 22 PM" src="https://github.com/No-57/SCSH_3C_iOS/assets/38306406/de6eb523-6b43-40ce-8bf5-afeb0c836aec">
 
-
+## CI
+### Prerequisites
+- Install Fastlane: https://docs.fastlane.tools/getting-started/ios/setup/
+- Set up Environment Variables: Environment variables are required by the CI script, Ensure that these are set up in your CI environment before running the script.
+  ``` console
+  // Create a file named `.env`
+  vi .env
+
+  // Add all the variables you need.
+  // ...
+
+  // export all variables on the terminal session.
+  source .env
+  ```
```

Comment:
```suggestion
  ```sh
  # Create a file named `.env`
  $ vi .env

  # Add all the variables you need.
  # ...

  # export all variables on the terminal session.
  $ source .env
  ```
```

Reply:
[done](https://github.com/No-57/SCSH_3C_iOS/pull/16/commits/4d7ee5454c8023e37eb2f7eaa285901e52110070)